clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,153 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 5}}
+{{$MEDIUM_BACKEND_SIZE := 3}}
+{{$SMALL_BACKEND_SIZE := 1}}
+{{$LARGE_BACKEND_LABEL := "ilblarge"}}
+{{$MEDIUM_BACKEND_LABEL := "ilbmedium"}}
+{{$SMALL_BACKEND_LABEL := "ilbsmall"}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$ilbQPS := 500}}
+{{$namespaces := 1}}
+#Test
+name: ilbload
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+- name: ILBConstantQPS
+  qpsLoad:
+    averageQps: {{$ilbQPS}}
+steps:
+# Create ILBs
+- measurements:
+  - Identifier: ServiceCreationLatencyLarge
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+      labelSelector: size = {{$LARGE_BACKEND_LABEL}}
+  - Identifier: ServiceCreationLatencyMedium
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+      labelSelector: size = {{$MEDIUM_BACKEND_LABEL}}
+  - Identifier: ServiceCreationLatencySmall
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+      labelSelector: size = {{$SMALL_BACKEND_LABEL}}
+  - Identifier: WaitForRunningDeployments
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: apps/v1
+      kind: Deployment
+      labelSelector: group = ilbload
+      operationTimeout: 15m
+- name: Creating ILBs
+  phases:
+  - namespaceRange:
+      min: 1
+      max: {{$namespaces}}
+    replicasPerNamespace: 1
+    tuningSet: ILBConstantQPS
+    objectBundle:
+    - basename: large-backends-dep
+      objectTemplatePath: dep.yaml
+      templateFillMap:
+        NumReplicas: {{$LARGE_BACKEND_SIZE}}
+    - basename: large-backends-service
```

Comment:
We should be creating service before creating deployment [best practices]

Reply:
Same below.

Reply:
Curious why this is.. by creating service first, potentially you can have the service VIP with no backends available yet, right?

Reply:
https://kubernetes.io/docs/concepts/configuration/overview/#services is the link